ndvi_forecast/1_download_data.ipynb

```diff
@@ -147,13 +148,14 @@
     "farmer_id = \"contoso_farmer\"\n",
     "try:\n",
     "    farmer = fb_client.farmers.get(farmer_id=farmer_id)\n",
-    "    print(\"Farmer {} Exists.\".format(farmer_id))\n",
+    "    print(\"Farmer '{}' exists.\".format(farmer_id))\n",
     "except ResourceNotFoundError:        \n",
-    "    print(\"Farmer doesn't exist...Creating ... \", end=\"\", flush=True)\n",
+    "    print(\"Farmer doesn't exist. Creating ... \", end=\"\", flush=True)\n",
```

Comment:
These are not consistent across diffrent resources eg farmer and boudnary. Lets fix one and use for all
Example:

'Farmer with id {} exists.'.format(farmer_id)
'Farmer with id {} does not exist. Creating ... '.format(farmer_id), end = ""
'Done'

This is just an example, please put whatever you feel is an appropriate description. But lets make sure that we have it same across all resources.

Reply:
Done